docs/coding.md

```diff
@@ -69,6 +69,34 @@ import (
 * Use log level `ERROR` when something occurs which is fatal to the operation,
   but not to the service or application.
 
+### Wrap long lines
+
+* At present, we restrict the number of chars in a line to `120` which is the
```

Comment:
no need for the bullet point or indention here, it is the only item in this section of the doc